crates/shim/src/mount.rs

```diff
@@ -615,6 +690,164 @@ pub fn mount_rootfs(
     Err(Error::Unimplemented("start".to_string()))
 }
 
+#[cfg(target_os = "linux")]
```

Comment:
I wonder if we should move linux specific code to `mount_linux.rs` ?
So we don't have to specify target for each function.